test/test_broker_app.py

```diff
@@ -49,7 +52,7 @@ def test_upload_success(self, mock_ingest, mock_async_upload, mock_request):
         # then
         self.assertEqual(response.status_code, 201)
         self.assertRegex(str(response.data), 'url/9')
-        mock_async_upload.assert_called_with('token', 'content'.encode(), False, None)
+        mock_async_upload.assert_called_with('token', 'content'.encode(), False, None, None)
```

Comment:
This line seems to repeat in several tests, I would suggest refactoring to a shared function.

Reply:
I decided to remove them, I think these assertions are brittle and too tied with implementation. Asserting the correct HTTP code and message match should be good enough.